test/servinge2e/helpers.go

```diff
@@ -6,13 +6,15 @@ import (
 	"testing"
 
 	"github.com/openshift-knative/serverless-operator/test"
+	corev1 "k8s.io/api/core/v1"
 	pkgTest "knative.dev/pkg/test"
 	"knative.dev/pkg/test/spoof"
+	servingv1 "knative.dev/serving/pkg/apis/serving/v1"
 	servingTest "knative.dev/serving/test"
 )
 
 const (
-	helloworldText = "Hello World!"
+	HelloworldText = "Hello World!"
```

Comment:
was defined in multiple places, so making it public here

Reply:
If we returned the same text as upstream we could avoid this.

Reply:
True, but separate topic.